README.md

```diff
@@ -1,3 +1,7 @@
+<div align="center" style="margin-bottom: 30px;">
```

Comment:
Thanks so much for your contribution @justinlawrence.

Can we just make this a little bit bigger and probably keep the title on top of the readme?

Probably something like this:

![image](https://user-images.githubusercontent.com/10208017/37708423-8560c5bc-2cfe-11e8-8b9b-2876e7f59069.png)

Reply:
650px looks neat